### PR TITLE
Docker creation

### PR DESCRIPTION
**Pull request by arielszabo**
Docker files creation.
Few notes:
- The whole build takes several minutes on my computer. if it's too long we can create the images
- Tasted on Ubuntu, need to be tested on Windows and Mac (shouldn't be a problem though)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -81,7 +81,7 @@ passlib==1.7.1
 pluggy==0.13.0
 promise==2.2.1
 protobuf==3.10.0
-psycopg2==2.8.3
+psycopg2-binary==2.8.3
 py==1.8.0
```
Comment (atalyaalon) [inline requirements.txt]: @arielszabo  why use the binary version? and why do you need to fix the version? (we just removed the fixed versions from the requirements.txt file)

Comment (arielszabo) [reply requirements.txt]: @atalyaalon I just didn't want to count on these packages to be backwards compatible so I fixed the version numbers  - your call if you want to change it. :)
The non-binary version just caused me some errors while building the docker.

Comment (atalyaalon) [reply requirements.txt]: for now, don't fix versions. regarding the binary package - that's fine.

Comment (arielszabo) [reply requirements.txt]: @atalyaalon Done  :)

**Pull request merged**